src/Commands/MakeUpgradeCommand.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Filament\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+class MakeUpgradeCommand extends Command
+{
+    use Concerns\CanManipulateFiles;
+
+    protected $description = 'Upgrade Filament to the latest version.';
+
+    protected $signature = 'make:filament-upgrade';
+
+    public function handle()
+    {
+        system("composer update");
+
+        system("php artisan migrate");
```

Comment:
Can we replace these calls with `$this->call` instead - it's definitely more reliable?

Reply:
Whoops only just seen this and I merged locally - will update.